lightspeed/modules/proc_login-vscode-extension.adoc

```diff
@@ -0,0 +1,26 @@
+:_content-type: PROCEDURE
+
+[id="login-vscode-extension_{context}"]
+= Logging in to Ansible Lightspeed through the Ansible VS Code extension
+
+After installing and configuring the VS Code extension, you can log in to the Ansible Lightspeed service.
+
+.Procedure
+
+. Open the VS Code application.
+. Use one of the following ways to connect to the Ansible Lightspeed service. 
+** Using the *Connect* button:
+.. From the navigation menu, click the *Ansible* icon. 
+.. Under Ansible Lightspeed Login, click *Connect*.
+** Using the *Accounts* button:
+.. From the navigation menu, click the *Accounts icon > Sign in with Ansible Lightspeed*.
++
+NOTE: This option is displayed when the VS Code extension is in an active state. The extension is activated after you open the Ansible side panel or after you open an Ansible file in the VS Code editor. If you do not see this option, use the *Connect* button to link to the Ansible Lightspeed service. 
+
+. When prompted, click *Allow* to sign in. 
+. On the *Authorize Ansible Lightspeed for VS Code* window, click *Authorize*.
```

Comment:
IBM uses "in the <blah, blah> window" as opposed to "on the window..." See one example [here](https://www.ibm.com/docs/en/ibm-style?topic=elements-mouse-buttons). I'll leave the decision to you.